jenkins/debian/debos/scripts/crush.sh

```diff
@@ -1,156 +1,202 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Why do you need `bash`?

If it's for the arrays when parsing the command line options, I think we can have a simpler implementation instead.

Reply:
Its not for arrays. After adding `function` Using `sh` reports error. Under `debos` container `/bin/sh` links to `dash` and running script as `sh crush.sh` fails. We need to run it as `bash crush.sh`  to avoid below error. 

`/tmp/script/crush.sh: Syntax error: "(" unexpected (expecting ";;")`

Reply:
The kernelci rootfs images are built with the KernelCI "base" Docker image, not the one from Debos.

Reply:
okay, then I'll use `sh` while sending the patch.